src/System.Reflection.TypeExtensions/tests/PropertyInfo/GetAccessors.cs

```diff
@@ -1,99 +0,0 @@
-// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-using System.Text;
-using Xunit;
-
-namespace System.Reflection.Tests
-{
-    public class Co4389GetAccessors
-    {
-        private static String s_strLoc = String.Empty;
-
-        /// data members corresponding to the properties
-        protected short m_prvPropAA = -1;
-        public short m_PropAA = -2;
-        public float m_PropBB = -2.0f;
-        private double _propCC = 123.456;
-
-        /// declare/define all the properties here
-
-        // MyPropAA - ReadWrite
-        private String MyPropAA
-        {
-            get { return m_PropAA.ToString(); }			//get accessor for property {String MyPropAA}
-            set { m_PropAA = Int16.Parse(value); }		//set accessor for property {String MyPropAA}
-        }
-
-        // MyPropBB - ReadWrite  property
-        public double MyPropBB
-        {
-            get { return (double)m_PropBB; }		//get accessor for property {double MyPropBB}
-            set { m_PropBB = (float)value; }		//set accessor for property {double MyPropBB}
-        }
-
-        private double MyPropCC
-        {
-            get { return (double)_propCC; }		//get accessor for property {double MyPropCC}
-            set { _propCC = (float)value; }		//set accessor for property {double MyPropCC}
-        }
-
-        [Fact]
-        public void GetAccessorsTests()
```

Comment:
The tests for GetAccessors() have been moved here: https://github.com/dotnet/corefx/pull/11510/files#diff-9d15e9462d42c76536a00157d9266ae5R53